### PR TITLE
removing inspector tab

### PR DESCRIPTION
**Pull request by martin-henz**
### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Code quality improvements

### Checklist

- [x] I have tested this code
- [ ] I have updated the documentation

### CODE REVIEW

**Comment by martin-henz** [comment]
The Steering Committee reached a consensus to remove the Inspector Tab. I took a look and discovered that the the inspector component in js-slang serves as tool to keep track of the line numbers when debugging. The "inspector" in cadet-frontend serves as controller: interface between editor and environment visualizer. 

This role of "inspector" makes sense, regardless whether we have an "Inspector" tab or not.

This PR therefore proposes to keep the "inspector" interface in js-slang and the controller component of "inspector" in cadet-frontend. 

The PR proposes just to remove the Inspector tab.

Any views welcome.

File: src/styles/_workspace.scss [hunk 1]
```diff
@@ -377,32 +377,6 @@ $code-color-error: #ff4444;
       }
     }
 
-    ##{$ns}-tab-panel_side-content-tabs_inspector {
```
Comment (samuelfangjw) [inline src/styles/_workspace.scss:380]: Removed unused styles

File: public/externalLibs/inspector/inspector.js [hunk 2]
```diff
@@ -1,225 +1,4 @@
 (function (exports) {
-  var container = document.createElement('div');
```
Comment (samuelfangjw) [inline public/externalLibs/inspector/inspector.js:2]: The functions I've removed don't seem to be referenced anymore. This means that this file (named inspector.js) is now only responsible for highlighting lines (when debugging).

**Review by samuelfangjw — COMMENTED** [review]
Looks good! I made some additional changes.

**Review by samuelfangjw — APPROVED** [review]

**Pull request merged**